src/js/components/Select/Select.js

```diff
@@ -278,19 +278,35 @@ const Select = forwardRef(
 
     const iconColor = getIconColor(theme);
 
+    let ariaLabelGenerated = '';
+    if (value) {
+      if (Array.isArray(value)) {
+        ariaLabelGenerated = `; ${value.length || 0} selected`;
```

Comment:
```suggestion
        ariaLabelGenerated = `; ${value.length || 0} ${
          value.length === 1 ? 'option' : 'options'
        } selected`;
```
Can we adjust this so it reads out `<x> options selected` instead of `<x> selected`. This helps clarify in the case where we might have options with numerical names.

Reply:
This sounds better. `SelectMultiple` component had `<x> selected` that's why I went with that. Maybe we can make another PR to make ariaLabel consistent in `SelectMultiple` also.

Reply:
Yes, good idea I think this change could be made in SelectMultiple too